### ISSUE TITLE
Error when running in a project.

### ISSUE THREAD

**Issue by VanshShah1** [acheong08/Bard#52]
I'm using Pynecone to build a build a web app and using Bard in it. It is giving me an error when I run it in the Pynecone project, but works perfectly in another file with just a command line application of Bard. I have thoroughly checked my code and there's no problem in the Pynecone section as other AI models are working fine in it. I think there's some problem in bard.
Error:

Process SpawnProcess-19:
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 76, in subprocess_started
    target(sockets=sockets)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/uvicorn/server.py", line 67, in serve
    config.load()
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/uvicorn/config.py", line 477, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/vanshshah/Documents/s.t.e.v.e. forge/neurum/neurum/neurum.py", line 7, in <module>
    chatbot = Chatbot(token)
              ^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/Bard.py", line 63, in __init__
    self.async_chatbot = self.loop.run_until_complete(
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 629, in run_until_complete
    self._check_running()
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 588, in _check_running
    raise RuntimeError('This event loop is already running')
RuntimeError: This event loop is already running
sys:1: RuntimeWarning:

coroutine 'AsyncChatbot.create' was never awaited

**Comment by VanshShah1** [comment]
Here's the complete code for the project:

from pcconfig import config
from Bard import Chatbot

import pynecone as pc

token = "BARD COOKIES(Updated it still not working)"
chatbot = Chatbot(token)


class State(pc.State):
    ans = ""

    def clicked(self):
        self.ans = str(chatbot.ask("hello")['content'])

def index() -> pc.Component:
    return pc.vstack(
        pc.heading("ai", color="white", font_size="5em"),
        pc.text(State.ans, color="white")
    )

styles = {
    "background_color": "black"
}


app = pc.App(state=State, style=styles)
app.add_page(index)
app.compile()

**Comment by acheong08** [comment]
Known issue: You are running with uvicorn which messes up **all** async projects. Here is a workaround: https://stackoverflow.com/questions/46827007/runtimeerror-this-event-loop-is-already-running-in-python

**Comment by acheong08** [comment]
tl;dr 
```py
import nest_asyncio
nest_asyncio.apply()
```

**Comment by VanshShah1** [comment]
Should I write this at the end of the code
import nest_asyncio
nest_asyncio.apply()

**Comment by VanshShah1** [comment]
OHHHHH it's fixed!!!! Thank you so much Antonio you're the true GOAT
